### PR TITLE
Add integration tests for naming service

### PR DESCRIPTION
**Pull request by Pantani**
closes https://github.com/trustwallet/blockatlas/issues/541

# Headline 

Add integrations tests for naming service domains

## Changes
- Move ens methods to `domains` package inside the `services` folder
- Create a package for `naming service` integrations tests

### CODE REVIEW

File: pkg/tests/integration/integration_test.go [hunk 1]
```diff
@@ -17,6 +17,9 @@ func Test(t *testing.T) {
 	} else {
 		config.LoadConfig(configPath)
 	}
-	ontology.TestOntology(t)
-	bitcoin.TestBitcoin(t)
+	platform.Init()
+
+	//ontology.TestOntology(t)
+	//bitcoin.TestBitcoin(t)
```
Comment (vikmeup) [inline pkg/tests/integration/integration_test.go]: any reason to disable?

File: services/domains/domains.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package domains
+
+import (
+	CoinType "github.com/trustwallet/blockatlas/coin"
```
Comment (vikmeup) [inline services/domains/domains.go]: did you consider rename coin to cointype ? to be `CoinType`

File: pkg/tests/integration/integration_test.go (hunk 1 above)
Comment (Pantani) [reply pkg/tests/integration/integration_test.go]: sorry I did forget, but it's already enabled now

File: services/domains/domains.go (hunk 2 above)
Comment (Pantani) [reply services/domains/domains.go]: no, this name comes from the current implementation, but I prefer to call this package `coins`, in the plural

**Pull request merged**